### PR TITLE
Translation For ops_compatibility.md

### PR DESCRIPTION
**Pull request by JupiterXXX**

### CODE REVIEW

**Comment by Mr-Linus** [comment]
@JupiterXXX the pr need to fix.

**Comment by JupiterXXX** [comment]
Have fixed issues above : )

File: site/zh-cn/lite/guide/ops_compatibility.md [hunk 1]
```diff
@@ -0,0 +1,1070 @@
...
+*   [tf.sqrt](https://www.tensorflow.org/api_docs/python/tf/sqrt)
+*   [tf.square](https://www.tensorflow.org/api_docs/python/tf/square)
+*   [tf.subtract](https://www.tensorflow.org/api_docs/python/tf/subtract)
+*   [tf.tile](https://www.tensorflow.org/api_docs/python/tf/tile)
+*   [tf.nn.batch_norm_with_global_normalization](https://www.tensorflow.org/api_docs/python/tf/nn/batch_norm_with_global_normalization)
+*   [tf.nn.bias_add](https://www.tensorflow.org/api_docs/python/tf/nn/bias_add)
+*   [tf.nn.fused_batch_norm](https://www.tensorflow.org/api_docs/python/tf/nn/fused_batch_norm)
+*   [tf.nn.relu](https://www.tensorflow.org/api_docs/python/tf/nn/relu)
+*   [tf.nn.relu6](https://www.tensorflow.org/api_docs/python/tf/nn/relu6)
+
```
Comment (Mr-Linus) [inline site/zh-cn/lite/guide/ops_compatibility.md]: 请参考 https://docs.google.com/document/d/1v7kwEhvtbbI4S4gulxLJDDN78gv0ZlDJ0cN92BUPrm0 修改，需要把文章中出现的 tensorflow 域名都需要改正。

Comment (JupiterXXX) [reply site/zh-cn/lite/guide/ops_compatibility.md]: 已经按要求替换域名: )

**Comment by lamberta** [comment]
Thank you.

Community translations are now located in the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) repo (see [this message](https://groups.google.com/a/tensorflow.org/g/docs/c/GJPq7S_fw4c/m/lA1qfBX2EQAJ) for context).

I'll move over/duplicate existing pull requests as I find time. In the meantime, please feel free to create a new pull request in that repo 😀 Then you can close this PR and add a link to the new one.

**Pull request closed without merging**